### PR TITLE
[EN] [PT-BR] [BR] Update Storyboards

### PR DESCRIPTION
**Pull request by pimpG**
LuigiHann's Walkie Talkie Man never had a SB in osu!, DS games don't have an option to disable SB as far as i know. also the "no SB" image is actually the  animation when playing decently.

### CODE REVIEW

File: wiki/Storyboards/en.md
Comment (TPGPL) [inline wiki/Storyboards/en.md:5]:
```suggestion
![This scene is shown while playing Walkie Talkie Man](img/Sbpassing.png "This scene is shown while playing Walkie Talkie Man")
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Comment by TicClick** [comment]
@pimpG are you able to find a reviewer for Portuguese?

**Comment by pimpG** [comment]
flantsune will review pt-br and pt-pt

**Comment by TicClick** [comment]
I don't think that really matters, anyone is welcome as long as they know the language, can proof-read the article, weed out translation mistakes, and make it not read like a machine translation by rewording unnatural/odd sentences

**Comment by TicClick** [comment]
I mean, even if there isn't a second pair of eyes, the worst that will happen is that your PR will be merged anyway, just after at least 2 weeks of existence, and with a GitHub-only visible warning that no one proofread it

File: wiki/Storyboards/pt-br.md
Comment (Flantsune) [inline wiki/Storyboards/pt-br.md:3]:
```suggestion
![Essa cena é mostrada enquanto joga Walkie Talkie Man.](img/Sbpassing.png "Essa cena é mostrada enquanto joga Walkie Talkie Man")
```
This extra part is not currently in the en.md in this PR
typos

File: wiki/Storyboards/pt-br.md
Comment (Flantsune) [inline wiki/Storyboards/pt-br.md:6]:
```suggestion
Uma **Storyboard** (conhecida como SB) é uma imagem animada que acompanha o [Beatmap](/wiki/Glossary), normalmente apenas para enfeitar, algumas vezes para mudanças na jogabilidade. Uma storyboard pode ser composta de praticamente qualquer coisa, mas geralmente consiste de efeitos visuais projetados para tornar o mapa mais visualmente agradável e distinto para os usuários. Storyboards são inspiradas em jogos como o [Osu! Tatakae! Ouendan](/wiki/Glossary#ouendan). Assim como sua inspiração, as storyboards se ajustam a performance do jogador e mostram coisas diferentes dependendo do quão bem o jogador está jogando.
```
Lacks the link to the Ouedan glossary, adjusted a full stop after it- The current one you have is fine too but don't omit the glossary link

File: wiki/Storyboards/pt.md
Comment (Flantsune) [inline wiki/Storyboards/pt.md:3]:
```suggestion
![Essa cena é mostrada enquanto joga Walkie Talkie Man.](img/Sbpassing.png "Essa cena é mostrada enquanto joga Walkie Talkie Man")
```
Same as in pt-br
This extra part is not currently in the en.md in this PR
typos, from what I could tell it is correct/exists in both Portuguese usages but not written like this in this context

File: wiki/Storyboards/pt.md
Comment (Flantsune) [inline wiki/Storyboards/pt.md:8]:
```suggestion
As storyboards são armazenadas em ficheiros de beatmap como [arquivos `.osb`](/wiki/osu!_File_Formats/Osb_(file_format)) autônomos ou extensões para a secção `[Events]` de um [arquivo `.osu`](/wiki/osu!_File_Formats/Osu_(file_format)). Por causa disso, é possível criar diferentes storyboards para diferentes mapas dentro de um [beatmapset](/wiki/Glossary#beatmapset).
```
pt-pt variant

File: wiki/Storyboards/pt.md
Comment (Flantsune) [inline wiki/Storyboards/pt.md:6]:
```suggestion
Uma **Storyboard** (conhecida como SB) é uma imagem animada que acompanha o [Beatmap](/wiki/Glossary), normalmente apenas para enfeitar, algumas vezes para mudanças na jogabilidade. Uma storyboard pode ser composta de praticamente qualquer coisa, mas geralmente consiste de efeitos visuais projetados para tornar o mapa mais visualmente agradável e distinto para os usuários. Storyboards são inspiradas em jogos como o [Osu! Tatakae! Ouendan](/wiki/Glossary#ouendan). Assim como sua inspiração, as storyboards se ajustam a performance do jogador e mostram coisas diferentes dependendo do quão bem o jogador está jogando.
```
Same as pt-br:
Lacks the link to the Ouedan glossary, adjusted a full stop after it- The current one you have is fine too but don't omit the glossary link

**Review by Flantsune — CHANGES_REQUESTED** [review]

File: wiki/Storyboards/pt.md
Comment (pimpG) [reply wiki/Storyboards/pt.md:3]: they were supposed to be there, i just couldn't get them to work as i wanted last time, but now i figured that the quotation marks were breaking the images so i'll just not use them anymore.

i'll fix both "enquanto" though

File: wiki/Storyboards/pt-br.md
Comment (pimpG) [reply wiki/Storyboards/pt-br.md:6]: i didn't omit the link, it was simply not there previously.

**Review by Flantsune — APPROVED** [review]
pt-br and pt-pt are good to go.
Note to maintainer: There's an additional line on en.md, and thus nl, ja, fr and de will be considered outdated to current en. If I'm not mistaking anything here, that is. pt-pt and pt-br both possess that new line already.

**Review by TicClick — APPROVED** [review]

**Pull request merged**
